clang/test/Driver/sycl-lto.cpp

```diff
@@ -4,6 +4,10 @@
 // RUN: not %clangxx -fsycl -foffload-lto=thin %s -### 2>&1 | FileCheck -check-prefix=CHECK_ERROR %s
 // CHECK_ERROR: unsupported option '-foffload-lto=thin' for target 'spir64-unknown-unknown'
 
+// Verify we error when using the new offload driver but with device code split set to off.
+// RUN: not %clangxx -fsycl --offload-new-driver -foffload-lto=thin -fsycl-device-code-split=off %s -### 2>&1 | FileCheck -check-prefix=CHECK_SPLIT_ERROR %s
+// CHECK_SPLIT_ERROR: unsupported option '-foffload-lto=thin' for target 'spir64-unknown-unknown'
```

Comment:
Seems like the error message will need to specify that the error is due to combined use of -foffload-lto=thin and -fsycl-device-code-split=off?

Reply:
Sure, I will add a better error, thanks.

Reply:
fixed in latest commit, thx